guides/common/modules/con_troubleshooting-virt-who.adoc

```diff
@@ -0,0 +1,6 @@
+[id="troubleshooting-virt-who_{context}"]
+= Troubleshooting virt-who
+
+You can troubleshoot virt-who by checking the service status, logs, and by identifying configuration issues.
+
+For more information, see link:https://docs.redhat.com/en/documentation/subscription_central/1-latest/html/getting_started_with_rhel_system_registration/adv-reg-rhel-config-vm-sub_#virt-who-troubleshooting-methods_[Virt-who troubleshooting methods] and link:https://docs.redhat.com/en/documentation/subscription_central/1-latest/html/getting_started_with_rhel_system_registration/adv-reg-rhel-config-vm-sub_#virt-who-troubleshooting-scenarios_[Virt-who troubleshooting scenarios] in _Getting Started with RHEL System Registration_ in the Subscription Central documentation.
```

Comment:
you can use https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L28 to shorten the links